src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -10,7 +10,7 @@
     "test:ci": "jest --coverage",
     "lint": "ng lint",
     "e2e": "ng e2e",
-    "makePretty": "pretty-quick --staged"
+    "makePretty": "pretty-quick"
```

Comment:
Not sure if you are talking about the script name or the plugin we use, so:
- We use `makePretty` so we know we are not using prettify directly. 
  But maybe calling it `pretty-quick` would make more sense.
  In the past this was a pre-commit git hook.
- We are using `pretty-quick` because it allow us to format just the files modified on your current branch, ignoring all the old files that are still not well formated and are irelevant to the PR. This plugin uses prettier.